docs-app/package.json

```diff
@@ -47,7 +48,7 @@
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
-    "ember-json-viewer": "*",
+    "ember-json-viewer": "workspace:*",
```

Comment:
This is the pnpm way to refer to a workspace package, see https://pnpm.io/workspaces#workspace-protocol-workspace